From f93a4f157e8d3b53d05907ac9764b1d261ae2f4a Mon Sep 17 00:00:00 2001
From: Jason Chen <b02280@freescale.com>
Date: Thu, 4 Aug 2011 13:18:46 +0800
Subject: [PATCH 0320/2508] ENGR00154108-1 fsl_devices.h: make ldb support two
 ipu in separate mode

make ldb support two ipu in separate mode

Signed-off-by: Jason Chen <b02280@freescale.com>
---
 include/linux/fsl_devices.h |   11 ++++++++---
 1 files changed, 8 insertions(+), 3 deletions(-)

diff --git a/include/linux/fsl_devices.h b/include/linux/fsl_devices.h
index 4309a70..b54458f 100644
--- a/include/linux/fsl_devices.h
+++ b/include/linux/fsl_devices.h
@@ -200,12 +200,17 @@ struct fsl_mxc_ldb_platform_data {
 #define LDB_SPL_DI1	2
 #define LDB_DUL_DI0	3
 #define LDB_DUL_DI1	4
-#define LDB_SIN_DI0	5
-#define LDB_SIN_DI1	6
-#define LDB_SEP		7
+#define LDB_SIN0	5
+#define LDB_SIN1	6
+#define LDB_SEP0	7
+#define LDB_SEP1	8
 	int mode;
 	int ipu_id;
 	int disp_id;
+
+	/*only work for separate mode*/
+	int sec_ipu_id;
+	int sec_disp_id;
 };
 
 struct fsl_mxc_lcd_platform_data {
-- 
1.7.7.4

